modules/toolbox/framework_tools.py

```diff
@@ -0,0 +1,25 @@
+# Helper functions for the whole framework to use
+
+import pandas as pd
+from sklearn import cross_validation
+from sklearn import preprocessing
+
+
+def get_train_test(path, filename):
+    pandas_data = pd.read_csv(path + filename, sep=';')
```

Comment:
Are we guaranteed the separator is always a ';'? 
If that is a requirement, it needs to be documented somewhere.
Also, `path` should be checked for validity.